### ISSUE TITLE
Wrong Sentiment 

### ISSUE THREAD

**Issue by x011** [garritfra/positive_hackernews#3]
Hi,
Some posts are incorrectly classified as positive, for example:

[U.S. Suicides Reached a Record High Last Year](https://www.wsj.com/health/healthcare/americans-suicide-highest-level-2022-02eb10ea)

**Comment by garritfra** [comment]
Oh, that one definitely shouldn't be in there. I'll see if the analysis can be tweaked a bit.

Thanks for reporting!

**Comment by x011** [comment]
Here's another one:

https://www.theguardian.com/world/2023/dec/14/ive-never-seen-anything-like-this-japan-says-reason-behind-1200-tonnes-of-fish-washing-ashore-is-unknown

**Comment by garritfra** [comment]
Hi, sorry that this has been on hold for so long. I'm currently doing some research about using a ML model for sentiment analysis. I believe this model is small and fast enough that it can be embedded into the GitHub Action used by this project:

https://huggingface.co/cardiffnlp/twitter-roberta-base-sentiment-latest?text=Covid+cases+are+increasing+fast%21

I'm currently away from my computer, but I'll give this a shot once I'm back home.

**Comment by garritfra** [comment]
In [df39ce3](https://github.com/garritfra/positive_hackernews/commit/df39ce3ab77d999c2cfc334f8ff57cfca6979617) I've switched from dictionary analysis to the said ML model. Both of your model exceed a .3 negative sentiment, which I set as the threshhold for the filter for now.

Sorry again that it took so long. I hope you're still using this feed. Either way, I'm happy this is finally more reliable. 🙂
